### PR TITLE
GH-#1154-changed var to nc_var

### PR DESCRIPTION
**Pull request by rahul799**
## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [ x] When possible
## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rahul799** [comment]
This PR adresses #1154

**Comment by rahul799** [comment]
@dlebauer Please let me know if you want any changes.

**Comment by mdietze** [comment]
```
R check of base/utils returned new problems:
checking R code for possible problems ... NOTE: pdf.stats: no visible global function definition for 'var'
Error: Please fix these and resubmit.
```

File: base/utils/R/utils.R [hunk 1]
```diff
@@ -310,7 +310,7 @@ pdf.stats <- function(distn, A, B) {
     f = ifelse(B > 2,
                B/(B - 2),
                mean(stats::rf(10000, A, B))))
-  var <- switch(distn,
+  nc_var <- switch(distn,
```
Comment (infotroph) [inline base/utils/R/utils.R]: This one shouldn't be renamed -- here `var` means "variance" not "netCDF variable". But this is a great illustration of why to switch the others to unambiguous names!

File: base/utils/R/utils.R [hunk 2]
```diff
@@ -324,7 +324,7 @@ pdf.stats <- function(distn, A, B) {
   ci <- qci(c(0.025, 0.975), A, B)
   lcl <- ci[1]
   ucl <- ci[2]
-  out <- unlist(list(mean = mean, var = var, lcl = lcl, ucl = ucl))
+  out <- unlist(list(mean = mean, nc_var = nc_var, lcl = lcl, ucl = ucl))
```
Comment (infotroph) [inline base/utils/R/utils.R]: keep named as `var` here too

File: base/utils/R/utils.R (hunk 1 above)
Comment (infotroph) [reply base/utils/R/utils.R]: But this change uncovered a different problem! The call to the `var` function below this on line 322 ought to be spelled with its namespace included, i.e. `var(stats::rf(1e+05, A, B))))` should be changed to `stats::var(stats::rf(1e+05, A, B))))`. The variable naming was keeping `R CMD check` from noticing that before.

Comment (rahul799) [reply base/utils/R/utils.R]: > This one shouldn't be renamed -- here `var` means "variance" not "netCDF variable". But this is a great illustration of why to switch the others to unambiguous names!

I wish I would have pointed this out.

**Comment by mdietze** [comment]
```
These files were changed by the build process:
 M base/utils/man/mstmipvar.Rd
Have you run devtools::check and commited any updated Roxygen outputs?
```

**Comment by rahul799** [comment]
@mdietze I have run the `make check` & `devtools::check` command, but I didn't get any new problems. I don't know why Travis is failing?

**Comment by mdietze** [comment]
Travis is still complaining about Roxygen changes to mstmipvar.Rd. I don't see any commits for that file, is your local build not detecting/generating changes?

**Comment by rahul799** [comment]
@mdietze Roxygen generated those changes, but it seems to be irrelevant & I Ignored them (which must not be done :smile: ). I have committed the change & I will take care of it next time.

**Pull request merged**